### ISSUE TITLE
SQL comments not showing in interface godocs

### ISSUE THREAD

**Issue by sbres** [kyleconroy/sqlc#1400]
### Version

1.11.0

### What happened?

When the go code is generated, the docstrings are written on the Querier interface methods.

This is annoying because the ide will check the docstring of the interface and not the one from the implementation.

### What operating system are you using?

macOS

### What database engines are you using?

PostgreSQL

### What type of code are you generating?

Go

**Comment by kyleconroy** [comment]
Can you please include a full reproduction of the issue?

**Comment by sbres** [comment]
I've created a repo with an example:
https://github.com/sbres/sqlc_example

The query code:
```
-- name: CreateNewUser :one
-- CreateNewUser will create a new user
INSERT INTO users ("name", "email", "password")
VALUES (
    $1, $2, $3
)
RETURNING id
;
```

The sqlc.yaml file config
```
version: "1"
packages:
  - name: "sqlc"
    path: "sqlc"
    queries: "query/"
    schema: "migration/"
    engine: "postgresql"
    emit_empty_slices: true
    emit_json_tags: true
    emit_interface: true
```

The generated method with the docstring:
```
// CheckUserExist will check if the user exist, retuns true if the exists
func (q *Queries) CheckUserExist(ctx context.Context, email string) (bool, error) {
	row := q.db.QueryRowContext(ctx, checkUserExist, email)
	var column_1 bool
	err := row.Scan(&column_1)
	return column_1, err
}
```

The interface without the doctring:
```
type Querier interface {
	CheckUserExist(ctx context.Context, email string) (bool, error)
	CreateNewUser(ctx context.Context, arg CreateNewUserParams) (int32, error)
}
```
The "issue" comes when using an IDE.

When I use the struct the IDE is able to find the docsting
```
func doSomethingStruct(q sqlc.Queries, ctx context.Context) {
	q.CheckUserExist(ctx, "email string")
}
```

<img width="663" alt="Screenshot 2022-01-31 at 12 32 03" src="https://user-images.githubusercontent.com/6924239/151794657-9bdd1113-9fe2-4691-9ad9-26e118648f5b.png">

But when I use the interface, the IDE can't find a docstring
```
func doSomethingInterface(q sqlc.Querier, ctx context.Context) {
	q.CheckUserExist(ctx, "email string")
}
```

<img width="709" alt="Screenshot 2022-01-31 at 12 31 49" src="https://user-images.githubusercontent.com/6924239/151794677-44abd68b-028e-43b1-af32-f89e12af8897.png">

This is maybe a feature more than a bug that could be added to improve the usability :)

**Comment by sbres** [comment]
Solved on #1458 🎉
